src/sourmash/cli/tax/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Define the command line interface for sourmash tax
+
+The top level CLI is defined in ../__init__.py. This module defines the CLI for
+`sourmash tax` operations.
+"""
+
+from . import summarize
+from . import classify
+from . import label
+from ..utils import command_list
+from argparse import SUPPRESS, RawDescriptionHelpFormatter
+import os
+import sys
+
+
+def subparser(subparsers):
+    subparser = subparsers.add_parser('tax', formatter_class=RawDescriptionHelpFormatter, usage=SUPPRESS, aliases=['taxonomy'])
+    desc = 'Operations\n'
+    clidir = os.path.dirname(__file__)
+    ops = command_list(clidir)
+    for subcmd in ops:
+        docstring = getattr(sys.modules[__name__], subcmd).__doc__
+        helpstring = 'sourmash tax {op:s} --help'.format(op=subcmd)
+        desc += '        {hs:33s} {ds:s}\n'.format(hs=helpstring, ds=docstring)
+    s = subparser.add_subparsers(
+        title='Integrate taxonomy information', dest='subcmd', metavar='subcmd', help=SUPPRESS,
```

Comment:
does this now get confusing in contrast to `lca`? Do we want to put `gather` in the description somewhere?

(soft suggestion, just asking)

I think this results in the following output when running `sourmash`:

```

    Integrate taxonomy information
        sourmash tax --help

```
in which case adding `based on 'gather' results` could be good. I'll put it in as a suggestion.